etc/orthography/WhiteHmong.tsv

```diff
@@ -1,193 +1,215 @@
+u	u	Y	White Hmong	146	ntug_dej;ntuj;hnub;hnub_qub;hnub_qub	U+0075
+^u	u	Y	White Hmong	1	us	U+005E U+0075
+-	+	_	White Hmong	1	ko-taw	U+002D
+_	+	_		32	pa_taws;lau_qaib;hma_liab;qe_ntshauv;rau_tes	U+005F
+v-	˧˦/³⁴ +	2 _		19	pov-txwv;hiav-txwv;ntxoov-ntxoo;qhov-muag;qhov-ntswg	U+0076 U+002D
+v_	˧˦/³⁴ +	2 _		147	hmoov_av;av_nkos;hmoov_zeb;qhov_tsua;nthwv_dej	U+0076 U+005F
+g-	˧˩/³¹ +	3 _		1	tsuag-tsuag	U+0067 U+002D
+g_	˧˩/³¹ +	3 _		56	ntug_dej;dej_ntaug_lawm;yawg_laug;pog_laug;pog_koob_yawg_koob	U+0067 U+005F
+j-	˥˧/⁵³ +	3 _		24	tiaj-nrag;tij-laug;luj-txwv;khauj-khaum;npuj-npaim	U+006A U+002D
+j_	˥˧/⁵³ +	3 _		100	ntiaj_teb;dej_ntaug_lawm;dej_nplawm;qhov_dej_txhawv;dej_poob_tsag	U+006A U+005F
+b-	˥/⁵ +	6 _		57	pob-tsuas;pob-zeb;kev_sib-nrauj;tub_xeeb-ntxwv;ntxhais_xeeb-ntxwv	U+0062 U+002D
+b_	˥/⁵ +	6 _		118	xuab_zeb;dej_poob_tsag;hnub_qub;xob_laim;xob_ntsais	U+0062 U+005F
+m_	˨˩˧/²¹³ +	6 _		72	nplaim_taws;niam_txiv;niam_laus;niam_hluas;menyuam_ntxaib	U+006D U+005F
+s_	˩/¹ +	6 _		101	npuas_dej;pas_dej;yis_dej;pos_huab;ntais_ntawv	U+0073 U+005F
+a$	a ˧/³³	A 4	White Hmong	16	pa;ncho_pa;kwv_tij_neej_tsa;hma;nab-qa	U+0061 U+0024
+ai$	ai ˧/³³	A 4		6	nplai;dai;sai;ntshai;kev_cai	U+0061 U+0069 U+0024
+au$	au ˧/³³	A 4		36	kab_taws_tsau;plaub-hau;plhaws_taub-hau;taub-hau;khauj-khaum_taub-hau	U+0061 U+0075 U+0024
+aw$	aɯ ˧/³³	A 4		11	ko-taw;yas_taw;luj_taw;hneev_taw;ntiv_taw	U+0061 U+0077 U+0024
+^vw	v ɯ	B I	White Hmong	1	vwm	U+005E U+0076 U+0077
+^cw	tɕ ɯ	C I	White Hmong	3	cwj_pwm;cwj_mem;cwj_nrig	U+005E U+0063 U+0077
+e$	e ˧/³³	E 4		16	qe;qhab-tse;nqaj-tse;nce;nce	U+0065 U+0024
+ee	e ŋ	E N	White Hmong	83	av_qeeg;neeg;txiv_neej;tub_xeeb-ntxwv;ntxhais_xeeb-ntxwv	U+0065 U+0065
+hw	h ɯ	H I	White Hmong	2	xib_hwb;xib_hwb	U+0068 U+0077
+^hw	h ɯ	H I	White Hmong	4	hwj-txwv;hwj_kais;hwj_txob;hwj	U+005E U+0068 U+0077
+i$	i ˧/³³	I 4		18	hli;nti;ci;ci;ncuav_ci	U+0069 U+0024
+w$	ɯ ˧/³³	I 4	White Hmong	1	ntxhw	U+0077 U+0024
+ws$	ɯ ˧/³³	I 4	White Hmong	3	ntsws;ntws;aws	U+0077 U+0073 U+0024
+ia	i/j ə	J E		128	ntiaj_teb;tiaj-nrag;hiav-txwv;hav_iav;pojniam	U+0069 U+0061
+^ia	i/j ə	J E		3	iav;iav;iab	U+005E U+0069 U+0061
+^kw	k ɯ	K I	White Hmong	7	kwv_tij;kwv;kwv_tij;kwv_tij_neej_tsa;kwj	U+005E U+006B U+0077
+^qw	q ɯ	K I	White Hmong	2	qwj;qw_nrov	U+005E U+0071 U+0077
+khw	kʰ ɯ	K I	White Hmong	1	thom_khwm	U+006B U+0068 U+0077
+^qhw	qʰ ɯ	K I	White Hmong	1	qhwv	U+005E U+0071 U+0068 U+0077
+lw	l ɯ	L I	White Hmong	2	muas_lwj;hauj_lwm	U+006C U+0077
+^lw	l ɯ	L I	White Hmong	4	lwg;lwj;lwm;lwm	U+005E U+006C U+0077
+^ml	m l	M L		4	mlom;mloog;mloog_lus;mlom	U+005E U+006D U+006C
+pw	p ɯ	P I	White Hmong	2	xub_pwg;cwj_pwm	U+0070 U+0077
+pli	pˡ i/j	P J		3	hauv_pliaj;caws_pliav;txiv_plig	U+0070 U+006C U+0069
+pl	p l	P L		3	tshaib_plab;tshuaj_pleev;tsev_hais_plaub	U+0070 U+006C
+^pl	p l	P L		20	plua_tshauv;plas;plaub-hau;plaub;plaub	U+005E U+0070 U+006C
+npl	ⁿp l	P L		12	dej_nplawm;hmoov_nplej;txiv_cev_nplaum;kaus_nplooj;chaw_ntaus_nplej	U+006E U+0070 U+006C

```

Comment:
Look @MacyL, this is not consequent: you use "pl" and render it as "p l", but above, you render "pli" as "p+superscript-l". How do we convert Ratliff2010 here? How do we convert p+l in Chen?